static/js/composebox_typeahead.js

```diff
@@ -181,6 +181,11 @@ function autocomplete_checks(q, char) {
         return false;  // char doesn't appear, or too far back
     }
 
+    // Don't autocomplete if there is a space following a '#'
+    if (char === "#" && q.charAt(last_at + 1) === " ") {
```

Comment:
string.charAt is defined to return the empty string if the index is out of range, but I can go ahead and add another test to be safe.